content/rules.md

```diff
@@ -10,6 +10,17 @@ These rules and the chosen format may change depending on the measures for the C
 
 # General Rules
 
+# Diversity Rules
+Each institution is permitted to send two teams by default. An institution can earn additional team slots under the following conditions:
+
+- The institution’s preliminaries consist of at least 30% female participants
+  - One additional slot awarded
+- The institution provides evidence of a genuine attempt to increase diversity at their preliminaries
+  - Maximum of 2 additional slots awarded
```

Comment:
Just a clarification question, is in addition to the slot awarded above (therefore a total of 3 additional can be earned) or in total, max 2 slots can be awarded ?